include/rocksdb/advanced_options.h

```diff
@@ -193,13 +193,13 @@ struct AdvancedColumnFamilyOptions {
   //
   // Setting this value to 0 will cause write buffers to be freed immediately
   // after they are flushed.
-  // If this value is set to -1, 'max_write_buffer_number' will be used.
+  // If this value is set to -1, 'max_write_buffer_number * write_buffer_size' will be used.
   //
   // Default:
   // If using a TransactionDB/OptimisticTransactionDB, the default value will
-  // be set to the value of 'max_write_buffer_number' if it is not explicitly
+  // be set to the value of 'max_write_buffer_number * write_buffer_size' if it is not explicitly
   // set by the user.  Otherwise, the default is 0.
-  int max_write_buffer_number_to_maintain = 0;
```

Comment:
Instead of removing, can we keep it as deprecated for backward compatibility reasons. We can sanitize the options to ensure that only one of ```max_write_buffer_number_to_maintain``` and ```max_write_buffer_size_to_maintain``` is specified.

Reply:
I feel supporting two options that do the same thing would complicate the logic and not easy to maintain in the long term. @siying is it OK to remove a CF option in a minor release?

Reply:
I think this one maybe OK, because it's a less used option. For a more used option, we may just keep it there to prevent users' build from broken. Please sync up with @yoshinorim about their plan of rolling it out.